iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/StaticVarCompensatorXml.java

```diff
@@ -41,8 +42,16 @@ protected boolean hasSubElements(StaticVarCompensator svc) {
     protected void writeRootElementAttributes(StaticVarCompensator svc, VoltageLevel vl, NetworkXmlWriterContext context) throws XMLStreamException {
         XmlUtil.writeDouble("bMin", svc.getBmin(), context.getWriter());
         XmlUtil.writeDouble("bMax", svc.getBmax(), context.getWriter());
-        XmlUtil.writeDouble("voltageSetPoint", svc.getVoltageSetPoint(), context.getWriter());
-        XmlUtil.writeDouble("reactivePowerSetPoint", svc.getReactivePowerSetPoint(), context.getWriter());
+
+        AtomicReference<String> voltageSetpointName = new AtomicReference<>("voltageSetpoint");
+        AtomicReference<String> reactivePowerSetpointName = new AtomicReference<>("reactivePowerSetpoint");
```

Comment:
Not sure why you are using `AtomicReference` here?

Reply:
Because we need to use final attribute in lamba expression, but if we don't have to change import and export, i'll have to revert this.

Reply:
Okay, I would rather use a `String[] voltageSetpoinName = new String[1];` in this case